src/modules/auth/credentials/BulkAssignmentContainer.js

```diff
@@ -0,0 +1,460 @@
+import React, { useState, useEffect, useMemo } from "react"
+import { useTranslation } from 'react-i18next';
+import { Link } from "react-router-dom";
+import { Card, CardBody, CardHeader,
+	CardFooter, Button, ButtonGroup } from "reactstrap";
+import {DataTable, ButtonWithAuthz } from 'asab-webui';
+import RoleDropdown from "../components/RoleDropdown";
+import { useSelector } from "react-redux";
+
+const BulkAssignmentContainer = (props) => {
+
+	const { t } = useTranslation();
+
+	const [data, setData] = useState([]);
+	const [count, setCount] = useState(0);
+	const [page, setPage] = useState(1);
+	const [limit, setLimit] = useState(0);
+	const [loading, setLoading] = useState(true);
+	const [credentialsFilter, setCredentialsFilter] = useState("");
+	const [selectedCredentials, setSelectedCredentials] = useState([]);
+
+	const [tenants, setTenants] = useState({});
+	const [tenantsPage, setTenantsPage] = useState(1);
+	const [tenantsCount, setTenantsCount] = useState(0);
+	const [tenantsLimit, setTenantsLimit] = useState(0);
+	const [tenantsFilter, setTenantsFilter] = useState("");
+	const [loadingTenants, setLoadingTenants] = useState(true);
+	const [selectedTenants, setSelectedTenants] = useState([{_id: 'Global roles', selectedRole: []}]);
+
+	const [show, setShow] = useState(false);
+	const [showTenantsContentLoader, setShowTenantsContentLoader] = useState(false);
+
+	let SeaCatAuthAPI = props.app.axiosCreate('seacat_auth');
+
+	const addResource = "authz:superuser";
```

Comment:
@petrKavulok add"what"resource ?

Reply:
I suggest to rename it just to `resource`, if you dont use any other resources within this container